### DISCUSSION TITLE
Add lifetime licence for Tiptap Pro

### DISCUSSION THREAD

**Discussion by AxelBriche** [ueberdosis/tiptap#2653] [Feature Requests]
Hi, I am currently not happy with the tiptap pro price for only 3 extensions (details, emoji and unique id), can you please add a lifetime licence with only one payment please?

29€ per month = 348€ / year, that's a lot of money for only 3 released extensions, and I'm not interested by uniq_id, incomplete video course (I want create complexe features, your video don't cover that). I have paid 129€ for tailwind ui who have hundreds of components, I don't want to pay 348€ per year for an access to 3 tiptap pro extensions source code. 

I primary want an access to tiptap pro for learn from the code source and create more complexe extension on my side (like spoiler or image gallery with upload features and interactions).

I want help and share my extension with the community, but I can't learn without difficulty how to use Tiptap for more complexe extensions, the documentation need more complexe examples.

**Comment by gustavo-bordin** [comment]
Indeed it sucks, super expensive, and not even going to use all the features.

**Reply by bdbch** [reply]
we pulled out a lot of extensions into our MIT-licensed open source base, so the only extensions still licensed under Pro are extensions only viable to use when you already use Tiptap services (Comments, Snapshots, AI).

Is there anything I miss here or what extension do you see that still is priced unfairly?

**Reply by vincentWuK** [reply]
I only need to use export docx feature, and i have 2 developers but gotta pay for 5 seats...... since CI/CD is counted as one developer..

**Comment by arnaugomez** [comment]
As of today, these 3 extensions (details, emoji and unique id) are open-source and MIT-licensed.

**Comment by bdbch** [comment]
As this post is from 2022 and we made big changes to our packages that were in the **Pro Licensing** before - I'll close this for now.

We moved tons of our previously paid packages to the open source repository and published the for everyone to use. Nowadays the only extensions that are on the paid plan are either extensions directly related to our cloud solutions like Tiptap Comments, Tiptap Snapshots, the conversion services, etc.
